bedrock/mozorg/urls.py

```diff
@@ -142,6 +142,7 @@
         name="mozorg.about.webvision.full",
     ),
     page("analytics-tests/", "mozorg/analytics-tests/ga-index.html"),
+    page("santalocator/", "mozorg/santa-locator.html"),
```

Comment:
Suggestion (non-blocking): would it be worth adding a redirect for `santa-locator` -> `santalocator` ?